API-Samples/C#/ArmBased/VideoIndexerClient/VideoIndexerClient.cs

```diff
@@ -95,36 +95,31 @@ public async Task<string> UploadUrl(string videoUrl , string videoName, string e
             }
 
             Console.WriteLine($"Video for account {_account.Properties.Id} is starting to upload.");
-            var content = new MultipartFormDataContent();
             
             try
             {
                 //Build Query Parameter Dictionary
                 var queryDictionary = new Dictionary<string, string>
                 {
-                    { "accessToken", _accountAccessToken },
                     { "name", videoName },
                     { "description", "video_description" },
                     { "privacy", "private" },
-                    { "partition", "partition" }
+                    { "accessToken" , _accountAccessToken },
+                    { "videoUrl" , videoUrl }
                 };
 
-                if (!string.IsNullOrEmpty(videoUrl) && Uri.IsWellFormedUriString(videoUrl, UriKind.Absolute))
-                {
-                    Console.WriteLine("Using public video url For upload.");
-                    queryDictionary.Add("videoUrl", videoUrl);
-                }
-                else
+                if (!Uri.IsWellFormedUriString(videoUrl, UriKind.Absolute))
                 {
                     throw new ArgumentException("VideoUrl or LocalVidePath are invalid");
                 }
+                
                 var queryParams = queryDictionary.CreateQueryString();
                 if (!string.IsNullOrEmpty(exludedAIs))
                     queryParams += AddExcludedAIs(exludedAIs);
 
                 // Send POST request
-                var url = $"{ApiEndpoint}/Accounts/{_account.Properties.Id}/Videos?{queryParams}";
-                var uploadRequestResult = await _httpClient.PostAsync(url, content);
+                var url = $"{ApiEndpoint}/westus2/Accounts/{_account.Properties.Id}/Videos?{queryParams}";

```

Comment:
Use `_account.Location` instead of `westus2`

Reply:
fixed